scripts/security/certs-create.sh

```diff
@@ -11,7 +11,16 @@ rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12 *.log
 # Generate CA key
 openssl req -new -x509 -keyout snakeoil-ca-1.key -out snakeoil-ca-1.crt -days 365 -subj '/CN=ca1.test.confluentdemo.io/OU=TEST/O=CONFLUENT/L=PaloAlto/ST=Ca/C=US' -passin pass:confluent -passout pass:confluent
 
-users=(kafka1 kafka2 client schemaregistry restproxy connect connectorSA controlcenter ksqlDBserver ksqlDBUser appSA badapp clientListen zookeeper mds)
+# ksqlDB Server (ksqldb-server) and Control Center (control-center) share a certificate; a separate certificate is not generated for ksqldb-server
+# this shared certificate allows the self-signed certificate to be accepted by a Control Center browser user visiting https://localhost:9092 ,
+# without importing and trusting the self-signed CA, and this acceptance will also apply later to Websockets requests to wss://localhost:8089
+# (port-forwarded to ksqldb-server:8089).  This is necessary as browsers never prompt to trust certificates for this kind of wss:// connection,
+# see https://stackoverflow.com/a/23036270/452210 .
+users=(kafka1 kafka2 client schemaregistry restproxy connect connectorSA control-center ksqlDBUser appSA badapp clientListen zookeeper mds)
```

Comment:
> ksqlDB Server (ksqldb-server) and Control Center (control-center) share a certificate

However, the certificate is called just control-center.  This is a touch confusing in the singular name, which could be alleviated if it is named to reflect its dual purpose.  Let's rename `control-center` to something that encompasses both names, e.g. `control-center-and-ksqldb-server`...WDYT?

Reply:
I can go either way with the naming - there's something nice about the consistency we are approaching, where each server requiring a cert has one with a filename and a DNS SAN matching its Docker hostname - but existing `mds` and now `control-center` and `ksqldb-server` have SAN script customisations, and now the latter pair are sharing files.  Happy to try with a shared filename.

Pushing two commits - the first just changes to a shared file-name e.g. `kafka.control-center-and-ksqldb-server.[truststore|keystore].jks`, replacing the per-service copies.

The second commit also replaces `control-center` with the common name `control-center-and-ksqldb-server` in the `users` generator list ... this necessitates more specific handling in the DNS SAN processing so that `control-center` can be retained as a valid name, since `$i` gets set to the CN and first DNS SAN.  As it stands at that commit, the common cert will have:

- `CN=control-center-and-ksqldb-server`, which won't match anything, but CN is less important these days
- DNS SAN: `[control-center-and-ksqldb-server, localhost, control-center, ksqldb-server]` ... again `control-center-and-ksqldb-server` is redundant and would require more script logic to remove it, but it's harmless.

@ybyzek please review and let me know what looks better to you.

Reply:
@javabrett  In this case, I think the ability to reuse the cert generation code is more beneficial than trying to create unique code paths for `control-center-and-ksqldb-server` (and `mds`, for that matter), so the code change LGTM.

I am in the process of running cp-demo with the latest commit at https://github.com/confluentinc/cp-demo/pull/279/commits/eb8cf3b114bcc289ebcb7e519cd2065fc9da58ed, and will report back if there is any friction.